collection-spec/README.md

```diff
@@ -1,4 +1,4 @@
-# STAC Collections
+# STAC Collection Specification
```

Comment:
renamed title to be consistent with other specs

Reply:
That's not correct. It's "Stac Items", "Stac Collections" and "STAC Catalogs" in RADME files. Only the ...-spec files and extensions have the "Specification" suffix. So I'd be in favour to revert this.

Reply:
> That's not correct. It's "Stac Items", "Stac Collections" and "STAC Catalogs" in README files. Only the ...-spec files and extensions have the "Specification" suffix. So I'd be in favour to revert this.

You're right, my mistake. reverted.